base/shell/explorer/traywnd.cpp

```diff
@@ -99,6 +99,35 @@ VOID RestoreWindowPos()
     g_WindowPosBackup.RemoveAll();
 }
 
+BOOL CanDialogSysMinimize(HWND hwnd)
+{
+    if (::GetClassLongPtrW(hwnd, GCW_ATOM) != reinterpret_cast<ULONG_PTR>(WC_DIALOG))
```

Comment:
Just using `(ULONG_PTR)WC_DIALOG` wouldn't work?

Reply:
In C++, C++-typecast is recommended. But we can use the simpler expression. OK.